react-strict.js

```diff
@@ -4,11 +4,17 @@ module.exports = {
     semi: 'error',
     complexity: 'error',
     curly: 'error',
-    'id-length': 'error',
+    'id-length': [
+      'error',
+      {
+        "exceptions": ["l", "e", "_"]
```

Comment:
Hey team, what do you think to also add as some exceptions `i`, `j` and `k`, for the use of nested for loops?? For sub-menus we're using a couple of nested loops to iterate the hierarchy.